app/models/sitemap_builder.rb

```diff
@@ -0,0 +1,35 @@
+class SitemapBuilder
```

Comment:
🐘 is this a service or a model?

Reply:
It was going to be a service, but then I had a good chat to josh about the semantics of service vs builder and because its a sitemap **builder** it was marginally more appropriate to make it a model.